target/linux/siflower/base-files/etc/inittab

```diff
@@ -0,0 +1,3 @@
+::sysinit:/etc/init.d/rcS S boot
```

Comment:
This file is the same as package/base-files/files/etc/inittab.

Reply:
Removed